### ISSUE TITLE
Blank screen

### ISSUE THREAD

**Issue by yaronyg** [radekstepan/burnchart#105]
I just followed the instructions, e.g. installed -g and ran burnchart 8080. When I open at localhost:8080 or 128.0.0.1:8080 I just get a blank screen. The source of the screen is:

```HTML
<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
  <link rel="stylesheet" type="text/css" href="css/app.bundle.css">
</head>
<body>
  <script src="js/app.bundle.js"></script>
</body>
</html>
```

I tried this on both Windows and Mac and got the same results.

**Comment by radekstepan** [comment]
Hi, haven't been keeping unminified versions of the app in the source, it should work now.

**Comment by radekstepan** [comment]
Published on NPM now as well.

**Comment by yaronyg** [comment]
That last part about publishing to NPM really helped the most. :)

Now if I can just figure out how to get it to make an estimate using the sizing. I put in labels, estimate - 2, estimate - 10, estimate -10. I changed the size_label to /^estimate - (\d+)$/ and I changed points to 'LABELS'. But http://127.0.0.1:8080/#thaliproject/Thali_CordovaPlugin/7 (which is a public project) shows nothing. I'm guessing this is because we haven't actually closed any issues yet for the milestone I'm looking at. But i was at least hoping it would give me something like "X points remaining".
